### PR TITLE
Try and fix erorrs with updated Envoy

### PR DESCRIPTION
**Pull request by ericvn**
**What this PR does / why we need it**:

Attempt to fix the failures in the pipeline

### CODE REVIEW

**Review by kyessenov — APPROVED** [review]

**Comment by kyessenov** [comment]
Alright, let me take a look.

File: test/integration/exchanged_token_integration_test.cc [hunk 1]
```diff
@@ -220,9 +220,10 @@ class ExchangedTokenIntegrationTest : public HttpProtocolIntegrationTest {
   }
 };
 
-INSTANTIATE_TEST_SUITE_P(Protocols, ExchangedTokenIntegrationTest,
-                         testing::ValuesIn(HttpProtocolIntegrationTest::getProtocolTestParams()),
-                         HttpProtocolIntegrationTest::protocolTestParamsToString);
+INSTANTIATE_TEST_SUITE_P(
+    Protocols, ExchangedTokenIntegrationTest,
+    testing::ValuesIn(HttpProtocolIntegrationTest::getProtocolTestParamsWithoutHTTP3()),
```
Comment (kyessenov) [inline test/integration/exchanged_token_integration_test.cc:225]: I don't think we should exclude h3?

Comment (kyessenov) [reply test/integration/exchanged_token_integration_test.cc:225]: The issue is that requests have ":authority" set to "host", so if you change them there, it'll pass with h3.

Comment (ericvn) [reply test/integration/exchanged_token_integration_test.cc]: Looks like the tests were passing (maybe an asan flake). Will try reverting the withoutH3 to see how that does,

**Pull request merged**